dataset/src/test/scala/frameless/functions/NonAggregateFunctionsTests.scala

```diff
@@ -17,6 +17,80 @@ class NonAggregateFunctionsTests extends TypedDatasetSuite {
     super.afterAll()
   }
 
+  test("sha2") {
+    val spark = session
+    import spark.implicits._
+
+    def prop(values:List[X1[Array[Byte]]])(implicit encX1:Encoder[X1[Array[Byte]]]) = {
+      Seq(224, 256, 384, 512).map(numBits => {
```

Comment:
I think `.map { numBits => ... }` is preferred over `.map(numBits =>  { ... })`

Reply:
point, I got lost in those brackets.